### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getGrammar' of undefined

### ISSUE THREAD

**Issue by pgetta** [TypeStrong/atom-typescript#269]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.192.0
**System**: Mac OS X 10.10.3
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package, v2.15.7

### Stack Trace

Uncaught TypeError: Cannot read property 'getGrammar' of undefined

```
At /Users/pavelgetta/.atom/packages/atom-typescript/dist/main/atomts.js:27

TypeError: Cannot read property 'getGrammar' of undefined
  at /Users/pavelgetta/.atom/packages/atom-typescript/dist/main/atomts.js:27:41
  at later (/Users/pavelgetta/.atom/packages/atom-typescript/dist/main/lang/utils.js:186:31)

```

### Commands

```
     -2:40.7 core:confirm (atom-text-editor.editor.mini)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "language-c",
      "language-java",
      "language-objective-c",
      "language-mustache",
      "linter-htmlhint"
    ],
    "themes": [
      "seti-ui",
      "greti-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.1.0
atom-typescript, v2.15.7
auto-update-packages, v0.2.2
autoclose-html, v0.15.0
autocomplete-css, v0.6.0
autocomplete-html, v0.5.0
autocomplete-plus, v2.10.0
autocomplete-snippets, v1.2.0
editorconfig, v0.3.3
emmet, v2.3.7
greti-syntax, v0.5.0
linter, v0.12.0
linter-csslint, v0.0.11
seti-ui, v0.6.3
tabs-to-spaces, v0.9.2
trailing-spaces, v0.2.5

# Dev
No dev packages
```

**Comment by pgetta** [comment]
issue happens just after installation

**Comment by pgetta** [comment]
and goes away after restarting atom.
By the way, there was one .ts file opened in atom while installing the plugin

**Comment by Bobris** [comment]
For me right after upgrade from 2.15.5 to 2.15.7 version of atom-typescript.

**Comment by basarat** [comment]
Sorry about that. Can't get a repro on windows but fix should work for mac :rose: Published : `v2.15.8`

**Comment by patrickjtoy** [comment]
Just got this issue when the upgrade to 2.15.2 finished installing. 

![image](https://cloud.githubusercontent.com/assets/5121391/7231930/95a2e03c-e746-11e4-9c0b-ca70bdfc84f3.png)

[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.192.0
**System**: Microsoft Windows 8.1 Pro
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package, v2.15.12

### Stack Trace

Uncaught TypeError: Cannot read property 'getGrammar' of undefined

```
At C:\Users\Patrick\.atom\packages\atom-typescript\dist\main\atomts.js:27

TypeError: Cannot read property 'getGrammar' of undefined
  at C:\Users\Patrick\.atom\packages\atom-typescript\dist\main\atomts.js:27:41
  at later (C:\Users\Patrick\.atom\packages\atom-typescript\dist\main\lang\utils.js:186:31)

```

### Commands

```
     -1:28.7 core:backspace (atom-text-editor.editor)
     -1:25.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor)
     -1:25.0 editor:newline (atom-text-editor.editor)
  2x -1:18.4 core:move-left (atom-text-editor.editor)
     -1:18.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor)
     -1:18.0 editor:newline (atom-text-editor.editor)
     -1:17.3 typescript:autocomplete (atom-text-editor.editor)
     -1:17.3 autocomplete-plus:activate (atom-text-editor.editor)
     -1:08.3 typescript:autocomplete (atom-text-editor.editor)
     -1:08.3 autocomplete-plus:activate (atom-text-editor.editor)
     -1:04.7 core:backspace (atom-text-editor.editor)
     -1:03.2 core:save (atom-text-editor.editor)
     -1:02.2 typescript:autocomplete (atom-text-editor.editor)
     -1:02.2 autocomplete-plus:activate (atom-text-editor.editor)
     -0:59.6 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-base16-ocean-dark-syntax.theme-one-dark-ui)
     -0:40.1 core:save (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "base16-ocean-dark-syntax"
    ],
    "projectHome": "C:\\Users\\Patrick\\Workspace\\APS2\\src"
  }
}
```

### Installed Packages

```coffee
# User
atom-typescript, v2.15.12
autocomplete-plus, v2.11.0
base16-ocean-dark-syntax, v1.0.1
emmet, v2.3.7
language-cshtml, v0.1.1
last-cursor-position, v0.8.0
linter, v0.12.0
minimap, v4.7.6
open-last-project, v0.4.1
pain-split, v1.3.1
project-manager, v1.15.5

# Dev
No dev packages
```
